src/bucketizers/index.ts

```diff
@@ -139,12 +151,24 @@ export class BucketizerOrchestrator {
                 const bucketizer = this.getBucketizer(i, prefix);
 
                 const getBucket = (value: string, root?: boolean) => {
-                    const terms = value.split("/");
-                    const key = encodeURIComponent(
-                        decodeURIComponent(terms[terms.length - 1]),
-                    );
+                    const encodedValue = value
```

Comment:
So if I understand correctly, we are now using the full id of the bucket instead of only the last part after the slash as key?
Just so I'm along with the change, what's the reason to change this?